### ISSUE TITLE
Clarification on Wright-Fisher Computation Equivalence

### ISSUE THREAD

**Issue by farzadsbakhtiar** [gravelLab/hiring_test#1]
An excerpt from somewhere in the middle of the notebook:

> In 2-, each parent can be chosen multiple times. This is equivalent to taking a sample from the previous generation, but with replacement. Convince yourself that this is true. This is not optional! **If you are not convinced**, try programming it both ways, or ask questions.

The question is, is *convinced* referring to the equivalence of *2-* and *sampling with replacement*, OR, equivalence of *1-* and *2-*.   
The way it's phrased, tells me that the former is the case, but it follows straight from definition and it made me wonder.

**Comment by sgravel** [comment]
You are correct -- if you are familiar with sampling and probability, this follows directly from the definition!
